### ISSUE TITLE
Very Very different PSNR between the original and the latest version

### ISSUE THREAD

**Issue by SUSTechWLA** [graphdeco-inria/gaussian-splatting#1048]
I have followed this program since Aug, 2023. (Many thanks to the relevant workers!) Recently, I noticed the program updated a lot. Therefore, I also updated the codes in my computer. However, when I trained the same directory, which is created by running COLMAP with a series of figures, I got a really very different PSNR after same times of training. With the original version, only use the loss function like belows,

Ll1 = l1_loss(image, gt_image)
loss = (1.0 - opt.lambda_dssim) * Ll1 + opt.lambda_dssim * (1.0 - ssim(image, gt_image))

the PSNR is aroud 20 or 30. However, with the latest version, loss funtion like,

Ll1 = l1_loss(image, gt_image)
if FUSED_SSIM_AVAILABLE:
  ssim_value = fused_ssim(image.unsqueeze(0), gt_image.unsqueeze(0))
else:
  ssim_value = ssim(image, gt_image)

loss = (1.0 - opt.lambda_dssim) * Ll1 + opt.lambda_dssim * (1.0 - ssim_value)

I got PSNR is aroud 7 or 8, less a lot. I have examined they have both the same training_report function, which print the PSNR. However, observed with my naked eyes that the difference between the figures was very tiny. High distance in PSNR number, low in figures. I want to know WHY? 

Really appreciate!!!

**Comment by 731875679** [comment]
I have found ANSWER.  They have different loadCam function in original code 
![image](https://github.com/user-attachments/assets/c1df8a1b-2904-4c47-88ec-3b4cacc58bea)
in PILtoTorch function normalizing the rgb from 0,255 - 0,1, but in latest version. The image is used directly. So we divide 255 when needing gt_image like belows, then we can get the same PSNR.
![image](https://github.com/user-attachments/assets/97b4e2af-7510-45e6-a919-38930bab46d9)

**Comment by AsherJingkongChen** [comment]
Please update your submodules by `git submodule update --init --recursive --remote`

See #1050

**Comment by SUSTechWLA** [comment]
> Please update your submodules by `git submodule update --init --recursive --remote`
> 
> See #1050

THANKS A LOT! I will try.

**Comment by SUSTechWLA** [comment]
> Please update your submodules by `git submodule update --init --recursive --remote`
> 
> See #1050

everything gonna be fine!!! you are right!!! really appreciate!!!

**Comment by scott198510** [comment]
> > Please update your submodules by `git submodule update --init --recursive --remote`
> > See [#1050](https://github.com/graphdeco-inria/gaussian-splatting/issues/1050)
> 
> everything gonna be fine!!! you are right!!! really appreciate!!!

@wla-98 
Which module in the submodule caused the above - mentioned problem due to the differences? diff-gaussian-rasterization？
